dvol_python/test_dvol.py

```diff
@@ -350,7 +350,6 @@ def test_non_standard_branch(self, volume_name, branch_name,
         self.assertEqual(
             [['*', volume_name, branch_name]], [line.split() for line in rest])
 
-    @skip_if_go_version
     def test_log(self):
```

Comment:
I agree, it would be beneficial and also simple to update the test, and now is the best time to do it